vm/sdk/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "athena-vm-sdk"
```

Comment:
TBH it is hard to follow and build a mental model of the whole thing with so many crates. Why are there 2 SDKs (`athena-sdk` and `athena-vm-sdk`)?. Also, there are `athena-lib`, `athena-vmlib`, `athena-vm`... Do you think that some of them could be combined?

The dependency graph (only the workspace):
![graph](https://github.com/user-attachments/assets/4eb4c4ec-c8f7-4bf9-ab1d-7d3071a0f341)

On the first glance, it seems that `athena-helper` and `athena-builder` could be combined easily.

Also, the `ffitest` crate could probably be structured as [integration tests](https://doc.rust-lang.org/book/ch11-03-test-organization.html#integration-tests) in the `athena-vmlib` create.

Reply:
> Why are there 2 SDKs (athena-sdk and athena-vm-sdk)?

The naming is confusing. One is an SDK used by developers writing Athena programs (guest side). One is an SDK used to fire up the Athena VM and run code (host side).

> there are athena-lib, athena-vmlib, athena-vm... Do you think that some of them could be combined?

`athena-vmlib` is pretty barebones and should be standalone since it's exposed via FFI, and contains all the messy FFI conversion code. I have to take another look at the others. I'm sure we can refactor things to make it cleaner, but we should do that after merging this PR.

The dependency graph is helpful, thanks.

Reply:
I created the graph with [cargo-depgraph](https://github.com/jplatte/cargo-depgraph): `cargo depgraph --workspace-only | dot -Tpng > deps.png`